at_client/lib/src/client/at_client_impl.dart

```diff
@@ -434,9 +433,9 @@ class AtClientImpl implements AtClient {
       String? sharedWith,
       bool isDedicated = false}) async {
     var builder = ScanVerbBuilder()
-      ..sharedWith = sharedWith
-      ..sharedBy = sharedBy
-      ..regex = regex
+      ..sharedWith = sharedWith!
+      ..sharedBy = sharedBy!
+      ..regex = regex!
```

Comment:
regex is optional  for scan.. regex! will throw runtime error if regex is not passed